pom.xml

```diff
@@ -3,9 +3,10 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.mesos</groupId>
-  <artifactId>storm</artifactId>
-  <version>0.9.6</version>
-  <name>storm</name>
+  <artifactId>storm-parent</artifactId>
+  <version>0.1.0</version>
```

Comment:
@erikdw I'm pro-releases vs pull from master and hope for the best.  I'm not sure how well the release plugin and build-release.sh (it greps the pom for version) will play together, likely we'll have to fake it.

I kicked around the idea of getting maven to do the storm download and tarball surgery. It's doable and would get the exact result you want.  I don't presently have time to do it, but could help review/advise.  Might be able to get me interest in January/February.  Should definitely be separate PR anyway.

P.S: will rebase and incorporate @choang this evening/tomorrow.  Do you want snapshot added to the version for now and will just mod it for releases?